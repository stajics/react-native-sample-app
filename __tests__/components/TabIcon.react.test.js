import 'react-native';
import React from 'react';
import TabIcon from '../../app/components/TabIcon.react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'; // eslint-disable-line

it('renders unselected tab icon', () => {
  const tabIcon = renderer.create(
    <TabIcon />
  ).toJSON();
  expect(tabIcon).toMatchSnapshot();
});

it('renders selected tab icon', () => {
  const tabIcon = renderer.create(
    <TabIcon
      selected
    />
  ).toJSON();
  expect(tabIcon).toMatchSnapshot();
});
