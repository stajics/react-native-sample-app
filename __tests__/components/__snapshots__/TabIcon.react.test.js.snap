exports[`test renders selected tab icon 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "borderColor": "black",
        "borderRadius": 25,
        "borderWidth": 1,
        "height": 50,
        "justifyContent": "center",
        "width": 50,
      },
      Object {
        "backgroundColor": "gray",
      },
    ]
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "white",
      }
    } />
</View>
`;

exports[`test renders unselected tab icon 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "borderColor": "black",
        "borderRadius": 25,
        "borderWidth": 1,
        "height": 50,
        "justifyContent": "center",
        "width": 50,
      },
      Object {
        "backgroundColor": "white",
      },
    ]
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
      }
    } />
</View>
`;
